packages/rocketchat-lib/server/startup/settings.js

```diff
@@ -4,7 +4,6 @@ import './email';
 // Insert server unique id if it doesn't exist
 RocketChat.settings.add('uniqueID', process.env.DEPLOYMENT_ID || Random.id(), {
 	public: true,
-	hidden: true,
```

Comment:
Will this update the current setting as well?

Reply:
yes.. I mentioned on the PR description.. no migration is required.. it will update the current one.